entrypoint.sh

```diff
@@ -157,7 +158,25 @@ main() {
   bindplane apply "$destination_path"
 
   echo "Applying configuration path: $configuration_path"
-  bindplane apply "$configuration_path"
+  bindplane apply "$configuration_path" > configuration.out
+  cat configuration.out
+
+  # When auto rollout is enabled
+  if [ "$enable_auto_rollout" = true ]; then
+    echo "Auto rollout enabled."
+    awk '{print $2}' < configuration.out | while IFS= read -r config
```

Comment:
This loops through each line in the file. The file looks like this:
```
Configuration k8s-cluster:1 created
Configuration k8s-gateway:1 created
Configuration k8s-node:1 created
```

Awk is used to grab the second column, which would be the config name with the version suffix. This allows us to do a `bindplane rollout status $config` for each configuration that was applied. This is better than checking all configs that might exist in bindplane, we only want to check on configs managed by the action.